app/src/main/AndroidManifest.xml

```diff
@@ -49,22 +49,28 @@
             android:theme="@style/AppTheme.NoActionBar" />
         <activity
             android:name=".activities.FirstUseActivity"
-            android:theme="@style/AppTheme.NoActionBar"
-            android:noHistory="true"/>
+            android:noHistory="true"
+            android:theme="@style/AppTheme.NoActionBar" />
         <activity
             android:name=".activities.CreateProfileActivity"
+            android:noHistory="true"
             android:theme="@style/AppTheme.NoActionBar"
-            android:windowSoftInputMode="stateVisible|adjustPan"
-            android:noHistory="true"/>
+            android:windowSoftInputMode="stateVisible|adjustPan" />
         <activity
             android:name=".activities.NFCActivity"
             android:theme="@style/AppTheme.NoActionBar" />
-        <activity
-            android:name=".activities.SelectVerificationMethodActivity" />
+        <activity android:name=".activities.SelectVerificationMethodActivity" />
+
+        <service android:name=".modules.pairing.AndroidBluetoothService" />
+
         <activity
             android:name=".activities.QRExchangeKeyActivity"
             android:theme="@style/AppTheme.NoActionBar" />
-
-        <service android:name=".modules.pairing.AndroidBluetoothService" />
+        <activity
+            android:name=".activities.SettingsActivity"
+            android:label="@string/title_activity_settings" />
+        <activity
+            android:name=".activities.KeyManagementActivity" />
```

Comment:
I think that was done automatically because I use "Create Activity" when you press on a package, which automatically adds it to the manifest. I will change it